src/plugins/vis_type_timelion/server/series_functions/es/lib/create_date_agg.js

```diff
@@ -29,29 +28,37 @@ export default function createDateAgg(config, tlConfig, scriptedFields) {
   };
 
   dateAgg.time_buckets.aggs = {};
-  _.each(config.metric, function (metric) {
-    metric = metric.split(':');
-    if (metric[0] === 'count') {
+  (config.metric || []).forEach((metric) => {
+    const metricBody = {};
+    const [metricName, metricArgs] = metric.split(/:(.+)/);
+    if (metricName === 'count') {
```

Comment:
I know that this is how it was before but do you want to use instead of `count` and `percentiles` the `METRIC_TYPES.COUNT` and `METRIC_TYPES.PERCENTILES` from the data plugin?

Reply:
Added